### PR TITLE
Fix issue with mysql 5.7.x + misc

### PR DESCRIPTION
**Pull request by ja-cea**

### CODE REVIEW

File: web_gui/acct/INSTALL [hunk 1]
```diff
@@ -1,2 +0,0 @@
-Requirements:
-httpd, php-mysql, php-xml, php, php-pdo, php-gd
```
Comment (tl-cea) [inline web_gui/acct/INSTALL]: These are no longer needed? If so, I'll have to update rbh install documentation.

Comment (tl-cea) [inline web_gui/acct/INSTALL]: Where is the duplicate of this file?

Comment (ja-cea) [inline web_gui/acct/INSTALL]: README.install in web_gui

**Comment by tl-cea** [comment]
Please also push to b_3.0 on gerrithub

**Pull request merged**